views/selectors.es

```diff
@@ -82,6 +82,14 @@ export const starCraftPlanSelector = createSelector(
     configSelector,
   ], config => _.get(config, 'plugin.poi-plugin-starcraft.plans', {})
 )
+export const equipAvailableSelector = createSelector(
+  [
+    equipsSelector,
+  ], equips => _(equips)
+    .filter({'api_locked': 0, 'api_level': 0})
```

Comment:
seems we could treat locked items available because one might lock them to avoid unintended scraping